src/dictionaries.py

```diff
@@ -35,10 +35,27 @@
 
 # Add a new waypoint to the list
 # YOUR CODE HERE
+waypoints.append({
+    "lat": 5,
+    "lon": -5,
+    "name": "a fourth place"
+})
+
 
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # YOUR CODE HERE
 
+waypoints[0].update({
```

Comment:
You can simply assign the value of 'name' for waypoint[0] to be equal to the new name without copying the rest of the key-value pairs